src/execution_plan/ops/op_sort.c

```diff
@@ -161,6 +162,26 @@ static OpResult SortReset(OpBase *ctx) {
 	return OP_OK;
 }
 
+static OpBase *SortClone(const ExecutionPlan *plan, const OpBase *opBase) {
+	assert(opBase->type == OPType_SORT);
```

Comment:
I think it would have been much nicer if we could simply call `NewSortOp` to create the clone...